src/lightning/data/streaming/cache.py

```diff
@@ -49,6 +50,7 @@ def __init__(
         chunk_size: Optional[int] = None,
         chunk_bytes: Optional[Union[int, str]] = None,
         item_loader: Optional[BaseItemLoader] = None,
+        max_cache_size: Optional[Union[int, str]] = "200GB",
```

Comment:
What does it mean if None? Could you add that to the docstring? Or if not needed, then we should remove `Optional`

Reply:
I removed the optional.